app/templates/frameworks/edit_declaration_section.html

```diff
@@ -52,11 +59,22 @@
               </div>
             {% endif %}
             {% for question in section.questions %}
-              {% if errors and (errors[question.id] or question.type == 'multiquestion') %}
-                {{ forms[question.type](question, declaration_answers, errors, question_number=question.number, get_question=get_question) }}
-              {% else %}
-                {{ forms[question.type](question, declaration_answers, {}, question_number=question.number, get_question=get_question) }}
-              {% endif %}
+                {{ forms[question.type](
+                    question,
+                    declaration_answers,
+                    errors if errors and (errors[question.id] or question.type == 'multiquestion') else {},
+                    question_number=question.number,
+                    get_question=get_question,
+                    message=(
+                            "This answer is from your {} declaration".format(name_of_framework_that_section_has_been_prefilled_from)
+                        if
+                            name_of_framework_that_section_has_been_prefilled_from and (declaration_answers.get(question.id) is not none or question.type == 'multiquestion')
```

Comment:
Additionally, `or question.type == 'multiquestion'` here seems to cause the "this answer is from your..." message to appear even on questions that have a missing previous entry as long as they're multiquestions.
![20170227missingprefillmultiquestions](https://cloud.githubusercontent.com/assets/807447/23370238/008f1078-fd0c-11e6-95ce-40c4fa2aa80c.png)